be/src/http/action/metrics_action.cpp

```diff
@@ -34,13 +34,14 @@ namespace doris {
 
 void MetricsAction::handle(HttpRequest* req) {
     const std::string& type = req->param("type");
+    const std::string& with_tablet = req->param("with_tablet");
```

Comment:
Need to update the doc `administrator-guide/operation/monitor-metrics/be-metrics.md`